packages/lit-dev-content/site/articles/article/lit-cheat-sheet.md

```diff
@@ -0,0 +1,1118 @@
+
+**Related Documentation & Topics:**
+
+- [Signals docs](/docs/data/signals/)
+- [Lit lifecycle](/docs/components/lifecycle/)
+- [`@lit-labs/signals` npm package](https://www.npmjs.com/package/@lit-labs/signals) (external)
+- [Signals standards proposal](https://github.com/tc39/proposal-signals) (external)
+
+### Pinpoint Signal Updates (watch directive)
+
+The `watch()` directive allows you to pinpoint exactly where a signal should
+update the DOM without re-triggering the lit-html re-render. What this means is
+that using the `watch()` directive will not trigger the `render()` unless it
+triggers the change of a traditional Lit Reactive Property.
+
+{% playground-ide "articles/lit-cheat-sheet/signals-watch-directive", true %}
+
+{% aside "warn" "no-header" %}
+
+This may be a helpful way to optimize performance in your Lit components, but
+*always measure for your use case*.
+
+{% endaside %}
+
+**Related Documentation & Topics:**
+
+- [The `watch()` directive](https://www.npmjs.com/package/@lit-labs/signals#watch-directive) (external)
+- [Reactive Properties](#reactive-properties)
+- [Signals docs](/docs/data/signals/)
+- [Lit lifecycle](/docs/components/lifecycle/)
+- [`@lit-labs/signals` npm package](https://www.npmjs.com/package/@lit-labs/signals) (external)
+- [Signals standards proposal](https://github.com/tc39/proposal-signals) (external)
+
+### Signals HTML Template Tag
+
+The `@lit-labs/signals` package also provides an `html` template tag that can
+be used in place of Lit's default `html` tag and automatically wraps any signals
+in the template with a `watch()` directive.
+
+{% playground-ide "articles/lit-cheat-sheet/signals-html-tag", true %}
+
+**Related Documentation & Topics:**
+
+- [The `html` tag and `withWatch()`](https://www.npmjs.com/package/@lit-labs/signals#html-tag-and-withwatch) (external)
+- [the `SignalWatcher()` mixin](#common-signal-setup-(signalwatcher))
+- [The `watch()` directive](#pinpoint-signal-updates-(watch-directive))
+
+### Make Signal Values from Other Signals (computed)
+
+Sometimes you need to derive a value from other signals. You can do this with
+a `computed()` signal.
+
+{% playground-ide "articles/lit-cheat-sheet/signals-computed", true %}
+
+**Related Documentation & Topics:**
+
+- [Signals docs](/docs/data/signals/)
+- [`@lit-labs/signals` npm package](https://www.npmjs.com/package/@lit-labs/signals) (external)
+
+### Reacting to Signal Changes (effects)
+
+The `@lit-labs/signals` package currently does not provide any signal-specific
+effects, but signal changes used in `render()` or the Lit lifecycle will trigger
+the Lit lifecycle like any other Reactive Property. You can trigger effects in
+the Lit lifecycle by using signals in the same way you would use Reactive
+Properties.
+
+{% playground-ide "articles/lit-cheat-sheet/signals-effect", true %}
+
+{% aside "warn" %}
+
+Effects implemented in this way are not compatible with the `watch()` directive.
```

Comment:
What does this mean? 

I think this section is confusing and should be limited to just referencing the effect in `signal-utils`.